File: C/Assignment 5/pyramid.c
// pyramid pattern
#include<stdio.h>
void main()
{
	int no=5;
	for(int row=1;row<=5;row++)
	{
		for(int space=1;space<=no-row;space++)
		    printf(" ");
		for(int star=1;star<=row;star++)
		    printf("* ");
	printf(" \n");
	}
}

File: OOP/Assignment 5/Pyramid.java
class Pyramid
{
public static void main(String args[])
{
	int no=5;
	for(int row=1;row<=5;row++)
	{
		for(int space=1;space<=no-row;space++)
		    System.out.printf(" ");
		for(int star=1;star<=row;star++)
		    System.out.printf("* ");
	System.out.printf(" \n");
	}
}
}